PalindromeLinkList

```diff
@@ -0,0 +1,75 @@
+//
```

Comment:
Thanks for uploading your written TEBOW IT notes!

[T] - Definitely good to clarify valid palindromes, which characters are allowed, and that the list is singly linked :) Be careful about making assumptions - the fact that all characters are acceptable data doesn't tell you if the function should be case-sensitive, so remember to ask your interviewer.

[E] - Good test cases :)

[B] - Thanks for leaving me the note :) This was a great approach for a doubly-linked list and it's good that you caught yourself. A space is actually a valid char (with its own value) and doesn't need to be treated specially.

[O/W] - Your optimization/walkthrough are solid. Thanks for drawing the diagrams and letting me know when you ran out of time.